opus/src/components/connection/conn.js

```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react';
+
+export default class conn extends Component {
```

Comment:
Even if this component isn't used yet, you can still keep conventions (explicit name & PascalCase)